setup.py

```diff
@@ -50,13 +50,18 @@
     )
     sys.exit(1)
 
+def set_builtin(name, value):
+    if isinstance(__builtins__, dict):
+        __builtins__[name] = value
+    else:
+        setattr(__builtins__, name, value)
```

Comment:
Can we please add a comment why this is necessary? E.g.
```suggestion
    else:
        # to support https://github.com/pypa/build
        # see https://github.com/jopohl/urh/issues/1106
        setattr(__builtins__, name, value)
```
IMO #1106  should be fixed in `build` itself but since it comes from PyPa I am fine keeping this patch in URH's codebase as long as there is a comment indicating why it is necessary.